### DISCUSSION TITLE
How to make Scrollbar Click-Area wider the visible bar?

### DISCUSSION THREAD

**Discussion by najak3d** [AvaloniaUI/Avalonia#7473] [Q&A] [answered]
Our desire is make the Scrollbar click-zone WIDER than it appears (especially needed for mobile touch-screens).

1. We want to have option of keeping the scroll "looking thin" (not getting wider on mouse-over)
BUT 
2. Making the click-zone as wide as we want. (a fat-finger width)

What is the best approach for doing this in Avalonia?

**Comment by kekekeks** [comment, marked as answer]
You can do that by replacing [ScrollBar's template](https://github.com/AvaloniaUI/Avalonia/blob/fed63630124dcdf4a2fcc10b1f9d0ca822d568f5/src/Avalonia.Themes.Default/Controls/ScrollBar.xaml#L1) to make its clickable parts to be mostly transparent (`Transparent` background is _not_ click-through) and using [ScrollViewer template](https://github.com/AvaloniaUI/Avalonia/blob/fed63630124dcdf4a2fcc10b1f9d0ca822d568f5/src/Avalonia.Themes.Default/Controls/ScrollViewer.xaml
) from Default Theme since it always uses a proper scrollback instead of auto-hiding it.

**Reply by najak3d** [reply]
Score!   That solution sounds like a winner to us.
